compiler/src/test/java/com/squareup/anvil/compiler/codegen/BindingModuleGeneratorTest.kt

```diff
@@ -429,7 +429,7 @@ class BindingModuleGeneratorTest(
     ) {
       assertThat(exitCode).isError()
 
-      assertThat(messages).contains("Source0.kt: (6, 11)")
+      assertThat(messages).contains("Source0.kt:6:11")
```

Comment:
Tedious change :/

Reply:
Aye, thanks @ZacSweers for handling the initial batch :). Needed to conditionally add back the old check now but at least using intellij find+replace with regex makes it pretty fast.